pkg/util/log/logpb/when_to_create_a_new_channel.md

```diff
@@ -0,0 +1,35 @@
+# When to create a new logging channel
+
+In general, we should avoid creating new logging channels.
+
+Additional logging channels increase the surface area of logging 
+configuration and introduce more points of confusion for customers 
+and operators in their deployments. We should strive to find ways 
+to satisfy logging needs through existing channels rather than 
+creating new ones.
+
+Here are some valid motivations for introducing new logging channels.
+Generally, it's recommended that your new channel fulfills at least 
+2 of these criteria before moving forward:
+
+1. There is a _collection_ of related logging activity that can be 
+   logically grouped separately from all other logs.
+   - For instance: `SQL_SCHEMA` is a channel that groups together 
+     all logs pertaining to schema changes. This is helpful because 
+     this information could otherwise be scattered across many 
+     different logs. 
+2. There is a strong need for _routing_ a subset of logging 
+   information differently from all other channels. (Note that 
+   logging infrastructure already provides tooling for 
+   differentiating between different sources or categorizations of 
+   logs.)
+   - For instance: `USER_ADMIN` and `PRIVILEGES` are likely to be 
```

Comment:
That's a good example for your point, but then the reader would immediately ask: Why two separate channels `USER_ADMIN` and `PRIVILEGES`? Doesn't look that they would be routed differently?

It's worth explaining that the reason those two in particular are separate from one another is because of point 3 below: we're expecting USER_ADMIN to out-crowd PRIVILEGES but we don't want to lose data from the latter due to file rotations.